client/my-sites/invites/controller.js

```diff
@@ -55,7 +56,15 @@ export function acceptInvite( context, next ) {
 				debug( 'Accepted invite for VIP sites' );
 				window.location.href = getRedirectAfterAccept( acceptedInvite );
 			} else {
-				const redirect = getRedirectAfterAccept( acceptedInvite );
+				const siteId = acceptedInvite.site.ID;
+				const isWPForTeamsSite = isSiteWPForTeams( context.store.getState(), siteId );
+
+				let redirect = getRedirectAfterAccept( acceptedInvite );
+
+				if ( isWPForTeamsSite ) {
```

Comment:
Would it be more clear if we do it inside `getRedirectAfterAccept`?

Reply:
Yep, I just thought it's better not to send in more stuff to that fn and "task" it to care about wp for teams sites but looking at it now it makes sense to centralize the behavior I guess.

Reply:
Refactored. Could you check it out once more pls?